src/plugins/kv-analytics-plugin.js

```diff
@@ -96,44 +98,69 @@ export default Vue => {
 
 			// Attempt Snowplow event
 			if (snowplowLoaded) {
-				// In case there is a problem with the tracking event ensure that the callback gets called after 500ms
-				let callbackCalled = false;
-				const callbackTimeout = setTimeout(() => {
-					if (!callbackCalled) {
-						callbackCalled = true;
-						callback({ error: 'timeout' });
-					}
-				}, 500);
-
-				// Snowplow API
-				/* eslint-disable max-len */
-				// https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/tracking-specific-events/#tracking-custom-structured-events
-				// https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/javascript-tracker/tracking-specific-events/#callback-after-track-2-15-0
-				/* eslint-eable max-len */
-				// snowplow('trackStructEvent', 'category', 'action', 'label', 'property', 'value', context, timestamp, afterTrack);
-				window.snowplow(
-					'trackStructEvent',
+				kvActions.trackSnowplowEvent({
```

Comment:
Created an independent event for this for use when re-playing events. While I've focused on snowplow for now we'll probably do the same for ga at some point.